src/main/java/Main.java

```diff
@@ -0,0 +1,117 @@
+import Documents.Book;
+import TFIDFCalculator.TFIDFCalculator;
+import Documents.Document;
+import FileReaders.TxtFileReader;
+import Normalizer.TokenNormalization;
+import SearchFilters.AndSearchFilter;
+import SearchFilters.NotSearchFilter;
+import SearchFilters.OrSearchFilter;
+import SearchQueryFilter.AndQueryHandler;
+import SearchQueryFilter.NotQueryHandler;
+import SearchQueryFilter.OrQueryHandler;
+import Stemmer.PorterStemmer;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Scanner;
+
+public class Main {
+
+    private final static String folderPath = "/Users/hosseinb/Desktop/SE_Team04_Summer1402/Phase01Project1/SoftwareBooksDataset";
+    private static ArrayList<Document> result = new ArrayList<>();
+    private static String query;
+    private static TokenNormalization normalizer;
+    private static PorterStemmer stemmer;
+    private static TFIDFCalculator tfidfCalculator;
+    private static ArrayList<Document> documents;
+
+    public static void getInput() {
+        System.out.println("Search query : ");
+        Scanner scanner = new Scanner(System.in);
```

Comment:
Scanner is never closed (could be a problem if we use `getInput()` more than once). Can be resolved using Lombok `@Cleanup`.